packages/frontend/core/src/modules/workbench/view/mobile/workbench-root.tsx

```diff
@@ -0,0 +1,32 @@
+import { useLiveData, useService } from '@toeverything/infra';
+import { useEffect } from 'react';
+import { type RouteObject, useLocation } from 'react-router-dom';
+
+import { WorkbenchService } from '../../services/workbench';
+import { useBindWorkbenchToBrowserRouter } from '../browser-adapter';
+import { ViewIslandRegistryProvider } from '../view-islands';
+import { ViewRoot } from '../view-root';
+
+export const MobileWorkbenchRoot = ({ routes }: { routes: RouteObject[] }) => {
```

Comment:
What if we just use desktop version WorkbenchRoot?

Reply:
move this file to `packages/frontend/mobile/src/modules/workbench/view/workbench-root.tsx` ?

Reply:
or `packages/frontend/mobile/src/pages/workspace/workbench-root.tsx`

Reply:
This is mobile-only, i think we should put it in @affine/mobile